### ISSUE TITLE
Change dark-on-light font color from black to dark gray (accessibility)

### ISSUE THREAD

**Issue by jeanchung** [chicagodart/chicagodart-theme#144]
The value of the $black variable in _variables.scss should be changed to dark gray, as straight black text on white blurs for some people with low vision.
